tests/unit/utils/MockDataForSVM.h

```diff
@@ -0,0 +1,83 @@
+#include <shogun/labels/BinaryLabels.h>
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/features/DataGenerator.h>
+
+using namespace shogun;
+
+class CMockDataForSVM
+{
+        public:
+                CMockDataForSVM() {/*Nothing to do in here*/}
+
+                ~CMockDataForSVM() {/*Nothing to do in here*/}
+
+                /**
+                 * This function is about to create features simpled from (2) gaussians distribution
```

Comment:
This function creates features and labels. (the kind of data should be described in the @brief doxygen field)